docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
```

Comment:
If you look at [the `onbuild` _Dockerfile_](https://github.com/docker-library/golang/blob/master/1.7/onbuild/Dockerfile), you can see that it copies the current directory tree into the directory _/go/src/app_ inside the container, then takes care of _go get_ and _go build_ for any dependencies of the copied source tree.

It seems strange here to fetch the _fargo_ source from GitHub when a developer interested in running tests is probably trying to check whether his _local changes_ broke anything—say, before submitting a PR.

Perhaps I misunderstand the intended usage of this image.